core/src/main/resources/AppConstants.properties

```diff
@@ -368,8 +368,16 @@ etag.ehcache.dir=${cache.dir}
 etag.ehcache.diskPersistent=false
 etag.ehcache.diskExpiryThreadIntervalSeconds=600
 
+# Below listed 4 suppression keys should be used with an adapter name to avoid losing any useful configuration warnings.
+# Example: warnings.suppress.transaction.{NameOfTheAdapter}
```

Comment:
What do you mean with 'to avoid _losing_ any useful configuration warnings'

Also maybe add : warnings.suppress.transaction.{NameOfTheAdapter}**=true**, as just specifying the property isn't enough.

Reply:
transaction suppression key suppresses 3 different warnings all related to transactions. It is possible that a programmer sets the property to true then do some modification in the configuration and forget to set it to false and lose some of the config warnings.

Reply:
Ah I understand what you mean. I think it's best to add an attribute to the enums where you can specify if it works on global or on adapter level. In the case of `warnings.suppress.transaction`, `warnings.suppress.integrityCheck` and `warnings.suppress.sqlInjections` you never want to set this on a global level (as it's too dangerous like you mentioned). A developer must specifically suppress a warning for a given adapter, so future (new) adapters will get these warnings again.